README.md

```diff
@@ -90,6 +91,41 @@ Joey -  It was our first time working with a backend team, we needed to discuss
 - user wants a wishlist or a planning to visit type of page
 
 
+## Part 2
+
+## 🖇️ Learning Goals
+### Backend Integration: 
+
+### Sprint Based development:
+
+### New Team Dyanmics with Backend Team:
+
+### Error Handling: 
+
+### Tailwind: 
+
+## 🎆 Wins && 🔥 Challenges#
+### Wins:
+Lex - This is a celebration of everything I have learned so far! I got to work with a backend team and learn more about what it takes to be a full stack dev.
+
+Joey - Getting the sort function to work was a massive win. I wanted it to be dynamic enough so that you could search for any park with any of the filter conditions provided. It turned out to be tougher of a task than I thought but the functionality of it turned out really nice.
+
+Arden - Wins would be finding useful libraries and documentation resources for our stretch, having strong collaboration within the frontend group, and integrating Tailwind quickly
+
+### Challenges: 
+Lex - pivoting from flutter and tailwind to just tailwind was disappointing but a needed step. I also felt greatly challenged by the weather modal because it had different rule than I had used before.
+
```

Comment:
I'll add my challenges in my branch since I'm still figuring out what to say for those.